File: devohda/javascript/PROGRAMMERS 42748.js
function solution(array, commands) {
    let answer = []
    answer = commands.map(command => {
        const [i, j, k] = command;
        const newArr = array.slice(i-1,j).sort((a,b) => {
            if(a > b) return 1;
            if(a === b) return 0;
            if(a < b) return -1;
        });

        return newArr[k-1];
    });

    return answer;
}

// javascript sort 는 예상과 다르게

File: devohda/python/PROGRAMMERS 42748.py
def solution(array, commands):
    answer = []
    for i, j, k in commands:
        new_arr = array[i - 1:j]
        new_arr.sort()
        answer.append(new_arr[k - 1])

    return answer

# 해당 코드로 풀었지만 ramda 를 사용한 코드를 보고 고쳐볼 예정
